### PR TITLE
By yanniboi: Updated composer to use Drupal 8 packages repo.

### PR DESCRIPTION
**Pull request by yanniboi**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,4 +23,4 @@
       "Drupal\\Tests\\decoupled_auth\\": "tests/src"
     }
   }
-}
+}
```
Comment (andrewbelcher) [inline composer.json]: Reporting an no newline at end of file error

File: composer.json [hunk 2]
```diff
@@ -6,12 +6,12 @@
...
     {
       "type": "composer",
-      "url": "https://packagist.drupal-composer.org"
+      "url": "https://packages.drupal.org/8"
     }
   ],
   "require": {
-    "drupal/profile": "8.1.*",
-    "drupal/address": "8.1.*"
+    "drupal/profile": "1.x-dev",
```
Comment (andrewbelcher) [inline composer.json]: Do we specifically need the dev branch, or is the latest alpha enough?

**Review by andrewbelcher — COMMENTED** [review]
Other than those 2 comments, yay for switching over!

**Comment by yanniboi** [comment]
@andrewbelcher updated

**Pull request merged**